static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -262,6 +262,7 @@ const StyledSidebarItem = styled(Link, {
   flex-shrink: 0;
   border-radius: ${p => p.theme.borderRadius};
   transition: none;
+  margin-bottom: 1px;
```

Comment:
damn this doesn't use flex where we can use `gap`?

Reply:
`SidebarSectionGroup`, but it uses `flex` not `grid`, which afaik is still not safe for `gap`

Reply:
https://github.com/getsentry/sentry/pull/47682/commits/9c3a2b22efcb948bc374b4b01437520240cd5560